draft-ietf-quic-transport.md

```diff
@@ -2256,130 +2297,150 @@ the server.
-|[Num Blocks(8)]|
+|                     Largest Acknowledged (i)                ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                Largest Acknowledged (8/16/32/64)            ...
+|                          ACK Delay (i)                      ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|        ACK Delay (16)         |
+|                       ACK Block Count (i)                   ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                     ACK Block Section (*)                   ...
+|                          ACK Blocks (*)                     ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~
 {: #ack-format title="ACK Frame Format"}
 
 The fields in the ACK frame are as follows:
 
-Num Blocks (opt):
-
-: An optional 8-bit unsigned value specifying the number of additional ACK
-  blocks (besides the required First ACK Block) in this ACK frame.  Only present
-  if the 'N' flag bit is 1.
-
 Largest Acknowledged:
 
-: A variable-sized unsigned value representing the largest packet number the
-  peer is acknowledging in this packet (typically the largest that the peer has
-  seen thus far.)
+: A variable-length integer representing the largest packet number the peer is
+  acknowledging; this is usually the largest packet number that the peer has
+  received prior to generating the ACK frame.
 
 ACK Delay:
 
-: The time from when the largest acknowledged packet, as indicated in the
-  Largest Acknowledged field, was received by this peer to when this ACK was
-  sent.
+: A variable-length integer including the time in microseconds that the largest
+  acknowledged packet, as indicated in the Largest Acknowledged field, was
+  received by this peer to when this ACK was sent.  The value of the ACK Delay
+  field is scaled by multiplying the encoded value by the 2 to the power of the
+  value of the `ack_delay_exponent` transport parameter set by the sender of the
+  ACK frame.  The `ack_delay_exponent` defaults to 3, or a multiplier of 8 (see
+  {{transport-parameter-definitions}}).  Scaling in this fashion allows for a
+  larger range of values with a shorter encoding at the cost of lower
+  resolution.
+
+ACK Block Count:
+
+: The number of Additional ACK Block (and Gap) fields after the First ACK Block.
 
-ACK Block Section:
+ACK Blocks:
 
 : Contains one or more blocks of packet numbers which have been successfully
   received, see {{ack-block-section}}.
 
 
 ### ACK Block Section {#ack-block-section}
 
-The ACK Block Section contains between one and 256 blocks of packet numbers
-which have been successfully received. If the Num Blocks field is absent, only
-the First ACK Block length is present in this section. Otherwise, the Num Blocks
-field indicates how many additional blocks follow the First ACK Block Length
-field.
+The ACK Block Section consists of alternating ACK Blocks and Gaps.  A First Ack
```

Comment:
I'd note that they're in descending order immediately, since it's been a source of confusion.  ie: "The ACK Block Section consists of alternating ACK Blocks and Gaps in descending packet number order."